packages/fiori/src/NotificationListItemBase.js

```diff
@@ -12,10 +12,10 @@ import "@ui5/webcomponents-icons/dist/message-warning.js";
 import "@ui5/webcomponents-icons/dist/overflow.js";
 
 // Templates
-import NotifactionOverflowActionsPopoverTemplate from "./generated/templates/NotifactionOverflowActionsPopoverTemplate.lit.js";
```

Comment:
Change "Notifaction" to "Notification" everywhere.

Reply:
Already done.